iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -784,6 +784,11 @@ public void removeAllBuses() {
             return states.get().calculatedBusBreakerTopology.getSwitchStream();
         }
 
+        @Override
+        public int getSwitchCount() {
```

Comment:
Are you really sure this view is the good one? NodeBreakerView instead?
By the way, your implementation is not really efficient: you should probably delegate to the calculatedBusBreakerTopology class to avoid a complete iteration.